stdout: âœ… Server listening on http://0.0.0.0:5001
, stderr: /usr/src/app/node_modules/mysql2/lib/promise/pool.js:36
    const localErr = new Error();
                     ^

Error: connect ECONNREFUSED 127.0.0.1:3306
    at PromisePool.query (/usr/src/app/node_modules/mysql2/lib/promise/pool.js:36:22)
    at ensureCols (/usr/src/app/server.js:37:30)
    at Object.<anonymous> (/usr/src/app/server.js:48:1)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'ECONNREFUSED',
  errno: -111,
  sql: undefined,
  sqlState: undefined,
  sqlMessage: undefined
}

Node.js v18.20.8
